nautobot/core/tests/test_views.py

```diff
@@ -71,6 +79,34 @@ def test_search(self):
         response = self.client.get(f"{url}?{urllib.parse.urlencode(params)}")
         self.assertHttpStatus(response, 200)
 
+    def test_appropriate_models_included_in_global_search(self):
+        # Check for existing models in core apps and example app
+        existing_models = []
+        core_app_list = ["circuits", "dcim", "extras", "ipam", "tenancy", "virtualization"]
```

Comment:
Could also use `apps.get_app_config("circuits")` (etc.) instead of directly importing `CircuitsConfig` etc. -- might allow for some code reduction?